### PR TITLE
adding tfm to host package reference

### PR DESCRIPTION
**Pull request by ramarag**
following up on https://github.com/dotnet/core-setup/pull/1728

@ericstj @gkhanna79 @eerhardt PTAL?

### CODE REVIEW

File: pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj [hunk 1]
```diff
@@ -66,6 +66,7 @@
     <!-- references the host packages -->
     <Dependency Include="Microsoft.NETCore.DotNetAppHost">
        <Version>$(AppHostFullVersion)</Version>
+      <TargetFramework>.NETCoreApp2.0</TargetFramework>
```
Comment (eerhardt) [inline pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: (Nit) is the white space lined up here?

Comment (gkhanna79) [inline pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: Do we want to have similar change for the other host packages as well?

Comment (ramarag) [inline pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: this is a temp fix will clean it up when i do the proper one

Comment (ramarag) [inline pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: @gkhanna79 basically every nuspec file only selects one group of dependecies. The way we group dependencies for M.N.App is tfm

**Pull request merged**